tecken/base/admin.py

```diff
@@ -73,3 +84,90 @@ def has_change_permission(self, request, obj=None):
+
+
+@login_required(login_url="/")
+@user_passes_test(lambda user: user.is_active and user.is_staff, login_url="/")
+def site_status(request):
+    context = {"settings": [], "versions": []}
+
+    # Figure out settings first
+    def clean_url(value):
+        parsed = urlparse(value)
+        if "@" in parsed.netloc:
+            # There might be a password in the netloc part.
+            # It's extremely unlikely but just in case we ever forget
+            # make sure it's never "exposed".
+            parts = list(parsed)
+            parts[1] = "user:xxxxxx@" + parts[1].split("@", 1)[1]
+            return urlunparse(parts)
+        return value
+
+    # Only include keys that can never be useful in security context.
+    keys = (
+        "ENABLE_AUTH0_BLOCKED_CHECK",
+        "ENABLE_TOKENS_AUTHENTICATION",
+        "ALLOW_UPLOAD_BY_DOWNLOAD_DOMAINS",
+        "DOWNLOAD_FILE_EXTENSIONS_ALLOWED",
+        "ENABLE_STORE_MISSING_SYMBOLS",
+    )
+    for key in keys:
+        value = getattr(settings, key)
+        context["settings"].append({"key": key, "value": value})
+
+    # Now for some oddballs
+    context["settings"].append(
+        {"key": "UPLOAD_DEFAULT_URL", "value": clean_url(settings.UPLOAD_DEFAULT_URL)}
+    )
+    context["settings"].append(
+        {
+            "key": "UPLOAD_TRY_SYMBOLS_URL",
+            "value": clean_url(settings.UPLOAD_TRY_SYMBOLS_URL),
+        }
+    )
+    context["settings"].append(
+        {
+            "key": "SYMBOL_URLS",
+            "value": json.dumps([clean_url(x) for x in settings.SYMBOL_URLS]),
+        }
+    )
+    context["settings"].append(
+        {
+            "key": "UPLOAD_URL_EXCEPTIONS",
+            "value": json.dumps(
+                {k: clean_url(v) for k, v in settings.UPLOAD_URL_EXCEPTIONS.items()}
+            ),
+        }
+    )
+    context["settings"].sort(key=lambda x: x["key"])
+
+    # Now get versions
+    with connection.cursor() as cursor:
+        cursor.execute("select version()")
+        row = cursor.fetchone()
+        (value,) = row
+        context["versions"].append(
+            {
+                "key": "PostgreSQL",
+                "value": value.split(" on ")[0].replace("PostgreSQL", "").strip(),
+            }
+        )
+    context["versions"].append(
+        {"key": "Tecken", "value": dockerflow_get_version(settings.BASE_DIR)}
+    )
+    context["versions"].append({"key": "Django", "value": get_version()})
+    redis_store_info = get_redis_connection("store").info()
+    context["versions"].append(
+        {"key": "Redis Store", "value": redis_store_info["redis_version"]}
+    )
+    try:
+        redis_cache_info = get_redis_connection("default").info()
+    except NotImplementedError:
+        redis_cache_info = {"redis_version": "fakeredis"}
+    context["versions"].append(
+        {"key": "Redis Cache", "value": redis_cache_info["redis_version"]}
+    )
+
+    context["versions"].sort(key=lambda x: x["key"])
```

Comment:
The one version this misses that was in the previous version is the version of React. I don't think it's all that helpful to know the React version and there are other ways to figure it out.